requirements/test.txt

```diff
@@ -3,3 +3,5 @@
 mock
 filelock
 pep8
+nox
```

Comment:
@jamesp Sorry... I missed this one. Did you want this as part of the `pip` requirements? I'm assuming not?

Reply:
gah!

Reply:
pushed update